sunpy/map/sources/sdo.py

```diff
@@ -208,5 +208,5 @@ def is_datasource_for(cls, data, header, **kwargs):
         Determines if header corresponds to an HMI synoptic map.
         """
         return (str(header.get('TELESCOP', '')).endswith('HMI') and
-                str(header.get('CONTENT', '')) ==
-                'Carrington Synoptic Chart Of Br Field')
+                'carrington synoptic chart' in
+                str(header.get('CONTENT', '')).lower())
```

Comment:
```suggestion
                'carrington synoptic chart' in str(header.get('CONTENT', '')).lower())
```
Don't see why it would need to be two lines (the original the same).